docs/Troubleshooing.md

```diff
@@ -0,0 +1,30 @@
+
+# Troubleshooting
+
+This help topic describes how to troubleshoot problems that may occur when you use the SurveyJS library in your application.
+
+## There is no visible page or question in the survey.
+
+**Issue**: The survey appears empty and the following message is displayed: "There is no visible page or question in the survey".
+
+**Solution**:
+You may encounter this issue in the following cases.
+
+* A survey JSON is bad-formatted.
+A survey model can be built from JSON. A survey JSON schema can be declared in your appplication, or imported from a file. 
+
+Validate JSON and ensure it is well-formatted. Also, ensure that the JSON schema follows the [survey structure](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#survey-structure) and declares the `elements` node at a minimum. For more information, refer to: [Define a Static Survey Model in JSON](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#define-a-static-survey-model-in-json).
+
+> NOTE: It is possible to programmatically build a survey model using the [addNewPage](https://surveyjs.io/form-library/documentation/surveymodel#addNewPage), [addNewPanel](https://surveyjs.io/form-library/documentation/pagemodel#addNewPanel), and [addNewQuestion](https://surveyjs.io/form-library/documentation/pagemodel#addNewQuestion) functions. For more details, refer to [Create or Change a Survey Model Dynamically](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#create-or-change-a-survey-model-dynamically).

```

Comment:
```suggestion
When you declare a survey JSON schema or load it from a file or a web server, ensure that it does not contain syntax errors (unclosed brackets or quotation marks, missing commas). Your JSON schema should also follow the [survey structure](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#survey-structure) and declare at least the `elements` property at the first level. Refer to the following help topic for more information: [Define a Static Survey Model in JSON](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#define-a-static-survey-model-in-json).

    > NOTE: Instead of a JSON schema, you can use methods to build a survey model. Refer to the following documentation article for details: [Create or Change a Survey Model Dynamically](https://surveyjs.io/form-library/documentation/design-survey-create-a-simple-survey#create-or-change-a-survey-model-dynamically).
```